Objects/longobject.c

```diff
@@ -4666,6 +4719,21 @@ long_lshift1(PyLongObject *a, Py_ssize_t wordshift, digit remshift)
         stwodigits x = m < 0 ? -(-m << remshift) : m << remshift;
         return _PyLong_FromSTwoDigits(x);
     }
+    if (a == DIGIT_ONE) {
```

Comment:
I'd be happier with a check that the _value_ of `a` is exactly `1`, instead of checking that `a` _is_ the small int `1`.

Reply:
Fixed.